docs/contributing/index.rst

```diff
@@ -40,52 +40,82 @@ Preperations for a new Python version
 RestrictedPython should be updated for each new version of Python.
 To do so:
 
--   Read the changelog (`What's new in Python`_).
--   Copy and adjust the new AST Grammar (found under: `Python 3 AST`_) to ``/docs/contributing/ast/python<version>.ast``.
--   Add a new file ``changes_from<old_version>to<new_version>.rst`` in the directory ``/docs/contributing/``.
--   Add those files to the ``toctree`` directive in ``index.rst``.
+* Read the changelog (`What's new in Python`_).
+* Create a pull request containing:
```

Comment:
The step to create a pull request should come last, and not have nested elements.